optuna/visualization/_parallel_coordinate.py

```diff
@@ -214,7 +214,7 @@ def _target(t: FrozenTrial) -> float:
             values = [math.log10(v) for v in values]
             min_value = min(values)
             max_value = max(values)
-            tickvals: list[int | float] = list(range(math.ceil(min_value), math.ceil(max_value)))
+            tickvals: list[int | float] = list(range(math.ceil(min_value), math.floor(max_value)))
```

Comment:
```suggestion
            tickvals: list[int | float] = list(range(math.ceil(min_value), math.floor(max_value) + 1))
```
Sorry, we needed to add 1 to the upper bound.